feat(commands): add Context.app_permissions and permissions as a shortcut to permissions_for self and author (#1429)
DisnakeDev/disnake: changelog/1429.feature.rst

> |commands| Add :attr:`.Context.app_permissions` and :attr:`.Context.permissions` as a shorthands synonymous with :attr:`.Interaction.app_permissions` and :attr:`.Interaction.permissions`.

Reviewer: ```suggestion
|commands| Add :attr:`.Context.app_permissions` and :attr:`.Context.permissions` as a shorthands similar to :attr:`.Interaction.app_permissions` and :attr:`.Interaction.permissions`.

```
